scripts/overlay/manual-mappings_test.go

```diff
@@ -79,54 +83,62 @@ func TestLoadManualMappings(t *testing.T) {
 		if mappings == nil {
 			t.Error("expected non-nil mappings")
 		}
-		if mappings == nil || len(mappings.Operations) != 5 {
-			t.Errorf("expected 5 operations, got %d", len(mappings.Operations))
+		if mappings == nil || len(mappings.Operations) != 6 {
+			t.Errorf("expected 6 operations, got %d", len(mappings.Operations))
+		}
+
+		// Verify enable operation
+		if mappings.Operations[0].Action != "enable" {
```

Comment:
Would it help to specify an enum for these indexes? It might make them feel less magical

Reply:
enum is a good idea, going to add that in